src/EntryPoints/HookHandlers.php

```
@@ -17,18 +28,52 @@ public static function onExtensionRegistration(): void {
 		SemanticWikibase::getGlobalInstance()->getFixedProperties()
 			->registerFixedTables( $customFixedProperties, $fixedPropertyTablePrefix );
 	}
 
 	public static function onSmwUpdateDataBefore( Store $store, SemanticData $semanticData ): void {
+		wfDebug( __METHOD__ . "SWB: onSmwUpdateDataBefore" );
 		SemanticWikibase::getGlobalInstance()->getSemanticDataUpdate()
 			->run( $semanticData );
+
 	}
 
+	public static function onPageSaveComplete( WikiPage $wikiPage, MediaWiki\User\UserIdentity $user, string $summary, int $flags, MediaWiki\Revision\RevisionRecord $revisionRecord, MediaWiki\Storage\EditResult $editResult ) {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$user'. (undefined)

(UnusedFormalParameter)